build.gradle

```diff
@@ -0,0 +1,130 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url = "https://oss.sonatype.org/content/repositories/snapshots/" }
+        jcenter()
+    }
+    dependencies {
+        classpath("com.github.jengelman.gradle.plugins:shadow:5.0.0")
+        classpath 'com.bmuschko:gradle-nexus-plugin:2.3.1'
```

Comment:
```suggestion
        classpath("com.bmuschko:gradle-nexus-plugin:2.3.1")
```

Reply:
Single quotes are used in most places, I think double quotes are better looking though so might wanna change the rest too